pkg/logs/message/message.go

```diff
@@ -5,25 +5,46 @@
 
 package message
```

Comment:
Not convinced by any of the refactoring bellow, can you describe your thoughts please ?